src/components/Loading/LoadingMessage/LoadingMessage.js

```diff
@@ -15,51 +23,41 @@ const LoadingMessage = ({
   small,
   withOverlay,
   ...other
-}) => {
-  const loading = (
+}) => (
+  <div
+    className={classnames(getComponentNamespace('loading-message'), className, {
+      [overlayNamespace]: withOverlay,
+      [`${overlayNamespace}--stop`]: withOverlay && !active,
+    })}
+  >
     <Loading active={active} small={small} withOverlay={false} {...other} />
```

Comment:
Did you notice that `withOverlay` is set to `false` here? 🤔 Was wondering why that is...?

Reply:
Sorry, I'm only seeing this now! I'm not entirely sure - I think @j1mie implemented this initially, but I believe it's to do with being able to position the message next to the `Loading` component correctly